impeller/typographer/backends/skia/text_frame_skia.cc

```diff
@@ -38,6 +39,11 @@ TextFrame TextFrameFromTextBlob(sk_sp<SkTextBlob> blob, Scalar scale) {
 
   for (SkTextBlobRunIterator run(blob.get()); !run.done(); run.next()) {
     TextRun text_run(ToFont(run.font(), scale));
+
+    // TODO(jonahwilliams): ask Skia for a public API to look this up.
```

Comment:
Can we file a bug for this and cross reference it here? We can followup.

Reply:
https://github.com/flutter/flutter/issues/112005

Reply:
Added bug to comment.